src/frontend/src/pages/DRCPage/DRCPage.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+import { useState } from 'react';
+import { Box, Grid, ListItemIcon, Menu, MenuItem, Typography, useTheme } from '@mui/material';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import { NERButton } from '../../components/NERButton';
+import NoteAddIcon from '@mui/icons-material/NoteAdd';
+import PageLayout from '../../components/PageLayout';
+import { DesignReview, DesignReviewStatus } from 'shared';
+import MonthSelector from './MonthSelector';
+import DayCard from './DayCard';
+import FillerCard from './FillerCard';
+import { batman } from '../../../../backend/tests/test-data/users.test-data';
+
+const DRCPage = () => {
+  const theme = useTheme();
+
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const [displayMonth, setDisplayMonth] = useState<Date>(new Date());
+
+  const EventDict = new Map<Number, DesignReview[]>();
+  // Test data:
+  EventDict.set(new Date().getDate(), [
+    {
```

Comment:
move this object to a design-review.utils.ts file and add a TODO comment to delete it for the wire up ticket